src/SelectableContext.tsx

```diff
@@ -1,14 +0,0 @@
-import * as React from 'react';
-import { SelectCallback } from './helpers';
-
-const SelectableContext = React.createContext<SelectCallback | null>(null);
-
-export const makeEventKey = (
-  eventKey?: string | number | null,
-  href: string | null = null,
-): string | null => {
-  if (eventKey != null) return String(eventKey);
-  return href || null;
-};
-
-export default SelectableContext;
```

Comment:
should these be re-exported? IDK if users are using this directly, probably they shouldn't

Reply:
Let's leave it out for now and see if we get any feedback about it being missing